packages/admin/cms-admin/src/warnings/WarningsGrid.tsx

```diff
@@ -83,12 +81,8 @@ export function WarningsGrid({ warningMessages: projectWarningMessages }: Warnin
         ...useDataGridRemote({ initialFilter: { items: [{ field: "state", operator: "is", value: "open" }] } }),
         ...usePersistentColumnState("WarningsGrid"),
     };
-    const warningMessages = { ...cometWarningMessages, ...projectWarningMessages };
-    const history = useHistory();
-    const { entityDependencyMap } = useDependenciesConfig();
-    const apolloClient = useApolloClient();
-    const contentScope = useContentScope();
-    const { values: scopeValues, createUrl } = useContentScope();
+    const { warningMessages } = useWarningConfig();
```

Comment:
The warningMessages prop from the grid now appears to be unused. Can we remove it?

Reply:
true sorry, forgot.
Done here:
 - 3abaeda144ae30eefbd233e9f8575e98b550bf16
 - 28176c51c1d1b4317fd31af640717a1583399bac